mlflow/R/mlflow/R/databricks-utils.R

```diff
@@ -77,10 +77,12 @@ get_databricks_config_from_env <- function() {
 get_databricks_config <- function(profile) {
   config <- if (!is.na(profile)) {
     get_databricks_config_for_profile(profile)
-  } else if (exists("spark.databricks.token") && exists("spark.databricks.api.url")) {
+  } else if (require("SparkR") &&
```

Comment:
`require("SparkR")` returns true if SparkR is installed. Obviously we expect this code to be run on databricks so `library('SparkR')` shouldn't be necessary.